sale_order_type/models/sale_order_type.py

```diff
@@ -34,10 +34,10 @@ def default_picking_policy(self):
         comodel_name='account.journal', string='Billing Journal',
         domain=[('type', '=', 'sale')])
     warehouse_id = fields.Many2one(
-        comodel_name='stock.warehouse', string='Warehouse', required=True)
+        comodel_name='stock.warehouse', string='Warehouse')
```

Comment:
Well, looking at the multi-company rule, company_id=False is allowed as a type seen by any company, so it's good to leave it empty and create a multi-company type.